app/src/main/res/layout/welcome_detail1.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
```RelativeLayout``` is outdated in the favour of ```ConstraintLayout```. ConstraintLayouts are extremely efficient in performance. If possible, convert it to ConstraintLayout.